frontend/src/modules/_shared/Figure.tsx

```diff
@@ -170,7 +169,14 @@ export class Figure {
     }
 
     makeLayout(): Partial<Layout> {
-        const layout = { ...this._plotLayout };
+        const layout = {
+            font: {
+                family: "Roboto, sans-serif",
+                size: 12,
+                color: "#333",
+            },
+            ...this._plotLayout,
```

Comment:
If you hardcode it like this, make a default layout config with a possibility to override it and do not have it directly in `makeLayout` as it prevents any customization.